corehq/messaging/scheduling/migrations/0001_initial.py

```diff
@@ -0,0 +1,148 @@
+                ('message', jsonfield.fields.JSONField(default=dict)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='IVRSurveyContent',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('form_unique_id', models.CharField(max_length=126)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='SMSContent',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('message', jsonfield.fields.JSONField(default=dict)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='SMSSurveyContent',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('form_unique_id', models.CharField(max_length=126)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='TimedEvent',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('order', models.IntegerField()),
+                ('day', models.IntegerField()),
+                ('time', models.TimeField()),
+                ('email_content', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='scheduling.EmailContent')),
+                ('ivr_survey_content', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='scheduling.IVRSurveyContent')),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='TimedSchedule',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('domain', models.CharField(db_index=True, max_length=126)),
+                ('include_descendant_locations', models.BooleanField(default=False)),
+                ('schedule_length', models.IntegerField()),
+                ('total_iterations', models.IntegerField()),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.AddField(
+            model_name='timedevent',
+            name='schedule',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='scheduling.TimedSchedule'),
+        ),
+        migrations.AddField(
+            model_name='timedevent',
```

Comment:
yeah, this is just a byproduct of using makemigrations. i don't think django tries to figure out which models have dependencies on each other when determining the order in which to create them, so if a model is created before a foreign key model is, the foreign key gets added later rather than with the initial create